crossterm_screen/src/screen/alternate.rs

```diff
@@ -75,6 +75,6 @@ impl AlternateScreen {
 impl Drop for AlternateScreen {
     /// This will switch back to the main screen on drop.
     fn drop(&mut self) {
-        self.to_main().unwrap();
+        let _ = self.to_main();
```

Comment:
But if we do this we swallow an error, is that correct behavior?

Reply:
The rule we're violating here is not to panic inside the `drop` fn. We have couple of options here ...

* Leave it as it is in this PR
* Add a new dependency ([log](https://crates.io/crates/log) crate) and use `error!` macro to log the error

... I'd say that we should proceed with the 2nd option. Swallowing errors is not a good practice as well.

Reply:
Imagine that you're user of this crate. You have lot of `Result`s everywhere, you're handling them and now you've got sudden panic and it's not your (user) fault. My plan is to remove all dangerous `unwrap()` calls, but not in this PR and not in the incoming release. Take it as a note what's in my queue.

Reply:
Okay although I don't like adding extra dependenies. I did an unwrap cleanup my self as well a while ago, and moved all unwraps to the top level api. When I did that I  already had made a lot of changes to the user api and decided to just panic on error and add result types in later releases. So that's were most of the library is at now. I will merge this PR